data/sponsors/philips.yml

```
@@ -0,0 +1,2 @@
+name: "Philips"
+url: "https://philips.nl/"
```

I realise that Philips hasn't sponsored a DOD event before, but might it make sense to set this to their generic `.com` site, then use a local override to point it to `.nl` for this event? Just trying to future-proof here (i.e. avoid having to rewrite history…).
```suggestion
url: "https://philips.com/"
```